src/ngraph/frontend/onnx_import/op/sub.hpp

```diff
@@ -48,8 +49,8 @@ namespace ngraph
             {
                 inline NodeVector sub(const Node& node)
                 {
-                    return {std::make_shared<ngraph::op::v1::Subtract>(node.get_ng_inputs().at(0),
-                                                                       node.get_ng_inputs().at(1))};
+                    return {std::make_shared<Subtract>(node.get_ng_inputs().at(0),
```

Comment:
This is example how we can use different opsets within one file.  Opset1 is included as `default_opset.hpp`.